alacritty/src/config/merge.rs

```diff
@@ -0,0 +1,85 @@
+//! Merge multiple configuration files together.
```

Comment:
This module should be renamed to something like `serde.rs`. Having to import `crate::config::merge::merge` makes little sense and doesn't allow for extension in the future.